src/admin-page/UsersTable.js

```diff
@@ -0,0 +1,245 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  IconButton, Input, Modal, Paper,
+  Table, TableBody, TableCell, TableContainer, TableHead, TableRow,
+} from '@material-ui/core';
+import { useMutation, useQuery } from '@apollo/client';
+import { PersonAdd, Done, Cancel } from '@material-ui/icons';
+import UserRow from './UserRow';
+import {
+  CREATE_USER, DELETE_USER, GET_USERS, UPDATE_USER,
+} from '../sources';
+import './styles.css';
+import UserModal from './UserModal';
+
+function getUserRoleId(userRoleName) {
```

Comment:
This could be written without ifs. 
`const userRoleMapping = { 'sys_admin': 1, 'company_user': 2, 'user': 3};`
and calling `userRoleMapping['sys_admin']` would return 1 etc. 
Since roles are used across the whole application, I would create a such constant in a utils/common place and export it.

Reply:
updated